src/track/beatfactory.h

```diff
@@ -19,21 +20,13 @@ class BeatFactory {
     static QString getPreferredVersion(const bool bEnableFixedTempoCorrection);
 
     static QString getPreferredSubVersion(
-            const bool bEnableFixedTempoCorrection,
-            const bool bEnableOffsetCorrection,
-            const int iMinBpm,
-            const int iMaxBpm,
             const QHash<QString, QString>& extraVersionInfo);
 
     static mixxx::BeatsPointer makePreferredBeats(const Track& track,
             const QVector<double>& beats,
             const QHash<QString, QString>& extraVersionInfo,
-            const bool bEnableFixedTempoCorrection,
-            const bool bEnableOffsetCorrection,
-            const int iSampleRate,
-            const int iTotalSamples,
-            const int iMinBpm,
-            const int iMaxBpm);
+            bool bEnableFixedTempoCorrection,
```

Comment:
IIRC we dropped the `b` prefix. 

```suggestion
            bool enableFixedTempoCorrection,
```

Reply:
`fixedTempoCorrectionEnabled` would be more consistent. You *enable* it only once and then it stays *enabled*. Similar to naming conventions for *commands* vs. *events* in event-driven systems.